trieste/acquisition/optimizer.py

```diff
@@ -54,8 +54,6 @@ def automatic_optimizer_selector(
     :return: The batch of points in ``space`` that maximises ``target_func``, with shape [1, D].
     """
 
-    optimizer: AcquisitionOptimizer[Box | DiscreteSearchSpace]
```

Comment:
Pylint flagged this as unnecessary. Should this be used elsewhere as a return type?

Reply:
I needed that to pass the type tests? If you can get rid of it, that would be great.

Reply:
Looks like it's no longer needed: I've removed it and the type tests are still passing.